examples/seismic/tutorials/13_LSRTM_acoustic.ipynb

```diff
@@ -0,0 +1,749 @@
+      "text/plain": [
+       "<Figure size 576x432 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "#NBVAL_IGNORE_OUTPUT\n",
+    "mask = Function(name='mask', grid=model.grid,space_order=0)\n",
+    "nztaper = 100\n",
+    "start=85\n",
+    "# the number of the taper samples are (nztaper-start)\n",
+    "for iz in range(nztaper):\n",
+    "    if iz <= start:\n",
+    "        mask.data[:,iz] = 1.\n",
+    "    else:\n",
+    "        mask.data[:,iz] = (1/(start+1-nztaper))*iz - nztaper*(1/(start+1-nztaper))\n",
+    "mask.data[:,:] = 1. - mask.data[:,:] \n",
+    "plot_image_mask(mask.data)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we are going to perform the LSRTM."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def lsrtm_gradient(nshots,mask,dm):\n",
```

Comment:
Not all variables are being passed as parameters, many of them being global, so nshots can also be global.